kvbc/include/kvbc_adapter/replica_adapter.hpp

```diff
@@ -223,6 +223,9 @@ class ReplicaBlockchain : public IBlocksDeleter,
   const concord::kvbc::categorization::KeyValueBlockchain *const getReadOnlyCategorizedBlockchain() {
     return kvbc_.get();
   }
+  std::map<std::string, std::vector<std::string>> getBlockStaleKeys(BlockId block_id) const {
```

Comment:
nit: These functions are in v1 also. Should we encapsulate these in an interface (IPrunerMetric) and then add them to v1, v4, and adapter.